quickcomm/migrations/0001_initial.py

```diff
@@ -0,0 +1,83 @@
+# Generated by Django 4.1.7 on 2023-02-22 20:04
```

Comment:
This file should probably not be included.

Reply:
I second that, let's not include migrations till the end.